src/services/accomodation.service.js

```diff
@@ -135,5 +135,57 @@ class accommodationService {
     });
     return query;
   }
+
+  /**
+ * creating rating query
+ * @param {string} rateData rate data.
+ * @returns {array} data the data to be returned.
+ */
+  static async CreateAccomodationRate(rateData) {
+    return Queries.create(db.accomodationRates, rateData);
+  }
+
+  /**
+ * creating rating query
+ * @param {integer} accomodationId accommodation id
+ * @param {integer} averageRate the rate average
+ * @returns {array} data the data to be returned.
+ */
+  static async updateAverageRate(accomodationId, averageRate) {
+    return Queries.updateAverageRate(db.accomodation, accomodationId, averageRate);
+  }
+
+  /**
+ * checking if the accommodation exist using it's id
+ * @param {integer} accomodationId accommodation id
+ * @returns {array} data the data to be returned.
+ */
+  static async checkAccommodationById(accomodationId) {
+    return Queries.checkAccommodationById(db.accomodation, accomodationId);
```

Comment:
@mujohn26 This function checkAccommodationExistance is the same as "checkAccommodationById" on line 172.

Reply:
They were not doing the same I changed it's name.